flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateTest.java

```diff
@@ -179,15 +181,17 @@ public void testBackwardsEventWithUninitializedChannel() throws Exception {
 
 		inputGate.setBufferPool(bufferPool);
 
+		final InputChannelMetrics metrics = new InputChannelMetrics(UnregisteredMetricGroups.createUnregisteredTaskMetricGroup().getIOMetricGroup());
```

Comment:
`InputChannelTestUtils.newUnregisteredInputChannelMetrics()`?

Reply:
fixed and squashed with last commit.